website/docs/eightshift-frontend-libs.md

```diff
@@ -9,11 +9,11 @@ This library is meant to bringing you the modern front end development tools to
 
 ## Prerequisites
 
-The knowledge of the core editor components and React.js is recommended since this library is focused on working with the new Gutenberg block editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
+The knowledge of the core editor components and React.js is recommended since this library is focused on working with the new Block editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```

Comment:
```suggestion
The knowledge of the core editor components and React.js is recommended, since this library is focused on working with the new Block editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```